mappings/58ad4a35e6e40d7e5833af259531dfb660d31b805166839d3fac305a4a45535553.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "58ad4a35e6e40d7e5833af259531dfb660d31b805166839d3fac305a4a45535553",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://jesuscrypt.io",
```

Comment:
Hi @JesusCrypt2024,

if your site is still not up, you can update this value, once it is running. The URL value is not required for the submission. For now, we can't verify the URL. Please, update the PR without URL value or with a valid page.

Thanks!

Reply:
Hi @VladislavKudrin,
The site has been published on the www clearly showing the Police ID on top (still some things under construction). 
I hope that fixes the problem, let me know any further actions. 

Thanks!